databags/sites.json

```diff
@@ -1,5 +1,13 @@
 {
   "site_list": [
+    {
+      "name": "MSD-LIVE",
+      "country": "US",
+      "description": "MSD-LIVE: a data and code management system combined with an advanced computing platform that enables MSD researchers to document and archive their data, run their models and analysis tools, and share their data, software, and multi-model workflows.",
+      "url": "https://data.msdlive.org",
+      "screenshot_filename": "msdlive.png",
+      "types": ["research-data", "repository"]
```

Comment:
The "repository" tag isn't used. If you want it to show up in the "Repository" facet, the tag is "institutional-repository"